src/main/java/Duke.java

```diff
@@ -1,10 +1,300 @@
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+/**
+ * Encapsulates the data and behaviour of the chatbot.
+ *
+ * @author Huang Zhuoyan, Celeste
+ * @version CS2103T AY24/25 Semester 1, G07
+ */
 public class Duke {
+    private static final String NAME = "Fatnom";
+    private static Storage storage;
+    private static final int LINE_LENGTH = 60;
+    private static DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+
+    /**
+     * Creates a line, in the form of a String.
+     *
+     * @return The line.
+     */
+    public static String createLine() {
+        String line = "";
+        for (int i = 0; i < LINE_LENGTH; i++) {
+            line += "_";
+        }
+        return line;
+    }
```

Comment:
Would it be better to just have a `private static final String = "_____";` instead of this method?